stan/math/gpu/kernels/helpers.hpp

```diff
@@ -32,6 +32,23 @@ std::string helpers =
   #ifndef dst
   #define dst(i,j) dst[j * dst_rows + i]
   #endif
+  #ifndef WORK_PER_WI_MULT
```

Comment:
Should we make a separate file for kernel constants?

Reply:
Eh, I think it's fine like this until this file becomes super huge. It's nice that developers looking at / working on kernels have only one file of predefined stuff to look at.